src/config/scrivitoSites.ts

```diff
@@ -37,12 +37,27 @@ export function siteForUrl(
 
 export async function ensureSiteIsPresent() {
   if ((await load(currentSiteId)) === null) {
-    navigateTo(() =>
-      Obj.onAllSites().where('_path', 'equals', '/').order('_language').first(),
-    )
+    navigateTo(() => {
+      const sites = Obj.onAllSites().where('_path', 'equals', '/').toArray()
```

Comment:
I'm not sure how `navigateTo` treats loadable content. Can you maybe explain?

Reply:
https://docs.scrivito.com/js-sdk/navigateTo#remarks

> Scrivito loads data asynchronously from the CMS backend. To make sure your target object data is available for navigation, pass in target as a function.

This means that navigation will happen only after all requested content has been fetched from the backend.
In this case these are all existing root objects.

Reply:
Thank your for the clarification!